lib/analytics.js

```diff
@@ -114,13 +126,18 @@ Analytics.prototype.init = Analytics.prototype.initialize = function(settings, o
     if (!Integration) delete settings[name];
   }, settings);
 
-  // add integrations
-  each(function(opts, name) {
-    var Integration = self.Integrations[name];
-    var integration = new Integration(clone(opts));
-    self.log('initialize %o - %o', name, opts);
-    self.add(integration);
-  }, settings);
+  if (doNotTrack) {
+    // Clear manually added integrations
+    this._integrations = {};
```

Comment:
ah okay. but the `msg.context.doNotTrack` being set on 692 would represent the value that the customer has chosen not the segment project owner. As long as inside the app we label this option "Respect browser preference to not be tracked" or something. It's just a bit confusing that the two options being set are the same name.

Also agree -- anytime we intro new context columns into warehouse without spec update or proper comms people freak out 😅 (rightfully so!) Speaking of spec, why do we have to name this so obscurely? We already have `context.device.adTrackingEnabled`.. can we name this like `context.doNotTrackWeb`?